networkx/convert.py

```diff
@@ -241,6 +242,7 @@ def from_dict_of_lists(d, create_using=None):
     return G
 
 
+# @nx._dispatch(...)  # Not implemented
 def to_dict_of_dicts(G, nodelist=None, edge_data=None):
     """Returns adjacency representation of graph as a dictionary of dictionaries.
```

Comment:
This function doesn't fit the patterns we're using. Here `edge_data=None` means to preserve all edge data, otherwise it indicates the edge _value_.